src/main/java/org/jabref/logic/formatter/casechanger/Word.java

```diff
@@ -18,20 +18,23 @@ public final class Word {
      */
     public static final Set<String> SMALLER_WORDS;
     public static final Set<Character> DASHES;
+    public static final Set<String> CONJUNCTIONS;
     private final char[] chars;
     private final boolean[] protectedChars;
 
     static {
         Set<String> smallerWords = new HashSet<>();
         Set<Character> dashes = new HashSet<>();
+        Set<String> conjunctions = new HashSet<>();
 
+        // Conjunctions
+        conjunctions.addAll(Arrays.asList("and", "but", "for", "nor", "or", "so", "yet"));
         // Articles
         smallerWords.addAll(Arrays.asList("a", "an", "the"));
         // Prepositions
         smallerWords.addAll(Arrays.asList("above", "about", "across", "against", "along", "among", "around", "at", "before", "behind", "below", "beneath", "beside", "between", "beyond", "by", "down", "during", "except", "for", "from", "in", "inside", "into", "like", "near", "of", "off", "on", "onto", "since", "to", "toward", "through", "under", "until", "up", "upon", "with", "within", "without"));
         // Conjunctions
-        smallerWords.addAll(Arrays.asList("and", "but", "for", "nor", "or", "so", "yet"));
-
+        smallerWords.addAll(conjunctions);
```

Comment:
In both comments it says 'Conjunctions'. Maybe change the comment add write one line, what difference it makes to conjuctions.

Reply:
Putting something like **_Conjunctions used as part of Title case capitalisation to specifically check if word is conjunction or not_** and **_Conjunctions used as part of all case capitalisation to check if it is a small word or not_** respectively  will be fine right?

Reply:
Yes. Would just be fine just to avoid misunderstanding.

Reply:
And don't forget the test case. Then we can merge

Reply:
Added test cases and modified the corresponding comments.